server/migrations/2020-07-18-234519_add_unique_community_user_actor_ids/up.sql

```diff
@@ -0,0 +1,50 @@
+-- Following this issue : https://github.com/LemmyNet/lemmy/issues/957
+
+-- Creating a unique changeme actor_id
+create or replace function generate_unique_changeme() 
+returns text language sql 
+as $$
+  select 'changeme_' || string_agg (substr('abcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyz0123456789', ceil (random() * 62)::integer, 1), '')
+  from generate_series(1, 20)
+$$;
+
+-- Need to delete the possible community and user dupes for ones that don't start with the fake one
+-- A few test inserts, to make sure this removes later dupes
+-- insert into community (name, title, category_id, creator_id) values ('testcom', 'another testcom', 1, 2);
```

Comment:
Probably remove this comment.

Reply:
I did use it a lot for testing, in some of the other migrations I left these lines in too. They make it easy to copy-paste when I was running and reverting the migration a bunch. I think its better to leave in just in case.